index.js

```diff
@@ -67,7 +67,7 @@ module.exports = options => {
 				insert(result, 'alias', alias, key);
 			});
 
-			if (props.default) {
+			if (Object.prototype.hasOwnProperty.call(props, 'default')) {
```

Comment:
If it also errors, I'll merge the existing code. That last option is just a bit shorter :)